extensions/amp-3q-player/0.1/test/test-amp-3q-player.js

```diff
@@ -0,0 +1,54 @@
+/**
```

Comment:
Could you please add `amp-3q-player` to the video-interface integration tests and make sure they pass?
To do so you need to:
1- Add a test to `test-video-players.js` (similar to amp-youtube that's already there)
2- Include the script in video-players.html

These will run an extensive suite of tests to ensure all the `postMessages` between the component and the iframe work properly. You can run the tests locally by running `gulp test --integration` 

If you run into test failures, best way to debug manually is to add `autoplay` attribute on your component in `examples/3q-player.amp.html` file and visit `localhost:8000/examples/3q-player.amp.max.html` (note the .max in the URL, it will ensure local code is used to run the example). 

Adding `autoplay` will instruct AMP runtime to use most of the `postMessage` video-interface integrations. Your video should start muted auto playing when it comes to view and pause when it goes out of view. Tapping the video should also unmute it.